### DISCUSSION TITLE
Recognize contributors and subprojects

### DISCUSSION THREAD

**Discussion by gbartolini** [cloudnative-pg/cloudnative-pg#3119] [Maintainers room]
Initially, we decided to open source CloudNativPG and adopt a simple [governance model based on maintainers](https://github.com/cloudnative-pg/cloudnative-pg/blob/main/GOVERNANCE.md). At that time we used the [related CNCF template as a base](https://github.com/cloudnative-pg/cloudnative-pg/blob/main/GOVERNANCE.md).

While this was the right choice at start of the project, it is proving to be inadequate for several reasons, primarily:

- the accelerated adoption of CloudNativePG
- the growth of the community
- the increasing number of satellite projects, like website, Helm charts, container images, and so on

The idea could be to move to the next stage, by partially using the [subproject template provided by the CNCF](https://github.com/cncf/project-template/blob/main/GOVERNANCE-subprojects.md) as a base and slowly integrate it in our governance model (not necessarily entirely - for example I don't think we are ready yet for elections). Subproject might need more autonomy and agility to make decisions indepedently, responsibly and in a faster way, without being slowed by the current maintainers' list.

We should also consider the contributor role (or similar) and properly [document it in the ladder page](https://github.com/cncf/project-template/blob/main/CONTRIBUTOR_LADDER.md).

I would like us maintainers to consider this.

**Comment by gbartolini** [comment]
Another interesting approach which, given our current growth phase, might be easier to achieve is the one proposed by Strimzi: https://github.com/strimzi/governance

**Comment by sxd** [comment]
The Strimzi approach it's quite more simple and basic, the one from the CNCF it's strict in a couple of rules and leave us some space to make our own model, which it's kind of nice.

I think we can properly take the general rules from CNCF model and apply some specific designs from the Strmizi model, what do you all think?

Regards!

**Reply by gbartolini** [reply]
I have proposed a few changes to our existing model: https://github.com/cloudnative-pg/governance/issues/1

**Comment by gbartolini** [comment]
See #4454
